skin-schema.json

```diff
@@ -512,6 +512,51 @@
         }
       }
     },
+    "skipControls": {
+      "description": "Toolbar which contains buttons that allow skipping back or forward and switching to the previous or next videos.",
+      "type": "object",
+      "required": [ "enabled", "skipBackwardTime", "skipForwardTime", "buttons" ],
+      "properties": {
+        "enabled": {
+          "description": "Show the \"Skip Controls\" toolbar if set to true, hide if set to false.",
+          "type": "boolean"
+        },
+        "skipBackwardTime": {
+          "description": "An integer number ranging from 1 to 60. Controls the amount of seconds that the playhead moves when the \"Skip Backward\" button is clicked.",
+          "type": "number",
+          "minimum": 1,
+          "maximum": 60
+        },
+        "skipForwardTime": {
+          "description": "An integer number ranging from 1 to 60. Controls the amount of seconds that the playhead moves when the \"Skip Forward\" button is clicked.",
+          "type": "number",
+          "minimum": 1,
+          "maximum": 60
+        },
+        "buttons": {
+          "description": "The collection of buttons that can be displayed on the \"Skip Controls\" toolbar.",
+          "type": "object",
+          "properties": {
+            "previousVideo": {
+              "description": "Button that allows switching to the last video that was played or to the previous video in a playlist.",
+              "$ref": "#/definitions/skipControlsButton"
+            },
+            "skipBackward": {
+              "description": "Button that allows skipping a few seconds backward in the video.",
```

Comment:
I would change it to "... allows skipping the video backward the amount of seconds defined in `skipBackwardTime`"

Same for `skipForward`

Reply:
Updated